packages/mc-scripts/src/config/create-webpack-config-for-production.ts

```diff
@@ -495,6 +495,14 @@ function createWebpackConfigForProduction(
           exclude: /node_modules/,
           use: [require.resolve('graphql-tag/loader')],
         },
+        process.env.ENABLE_MESSAGE_LOADER === 'true' && {
```

Comment:
Nit: let's name this a bit more specifically.

```suggestion
        process.env.ENABLE_WEBPACK_LOADER_I18N_MESSAGE_COMPILATION === 'true' && {
```